### PR TITLE
Network Observability 1.7 regenerate flows format doc

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

https://issues.redhat.com/browse/OSDOCS-11084
Issue:

Link to docs preview:

Merge only to `no-1.7`. On GA 10/21/24, Sara will open a branch to integrate `no-1.7` with `main`, and that branch will go to 4.12+.

NOTE to reviewers: The Vale warnings are n/a here because the text its calling out are formatting values, not regular text in a paragraph or step. 
QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by skrthomas** [comment]
@mehul one more OTEL piece to ACK. Needed to regenerate the Flow Format reference to include OTEL formats.

**Review by jpinsonneau — APPROVED** [review]
Looks good ! Thanks @skrthomas

**Review by memodi — APPROVED** [review]
/label qe-approved

**Comment by skrthomas** [comment]
Regarding these Vale errors, they are n/a because the values (k8s and dns) are part of a format. They aren't just regular text.

File: modules/network-observability-flows-format.adoc [hunk 1]
```diff
@@ -9,140 +9,162 @@ The "Filter ID" column shows which related name to use when defining Quick Filte
 
 The "Loki label" column is useful when querying Loki directly: label fields need to be selected using link:https://grafana.com/docs/loki/latest/logql/log_queries/#log-stream-selector[stream selectors].
 
-The "Cardinality" column gives information about the implied metric cardinality if this field was to be used as a Prometheus label with the `FlowMetric` API. For more information, see the "FlowMetric API reference". 
+The "Cardinality" column gives information about the implied metric cardinality if this field was to be used as a Prometheus label with the `FlowMetrics` API. Refer to the `FlowMetrics` documentation for more information on using this API.
```
Comment (lahinson) [inline modules/network-observability-flows-format.adoc:12]:
```suggestion
The "Cardinality" column contains information about the implied metric cardinality if this field was to be used as a Prometheus label with the `FlowMetrics` API. For more information about that API, see the `FlowMetrics` documentation.
```
The OCP documentation guidelines seem to prefer the "For more information, see ..." wording for pointing to other content.

**Review by lahinson — COMMENTED** [review]
Just a couple of suggestions. Looks good overall.

File: modules/network-observability-flows-format.adoc (hunk 1 above)
Comment (skrthomas) [reply modules/network-observability-flows-format.adoc:12]: Good nits, thank you!

**Pull request merged**